storagenode/bandwidth/service_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package bandwidth_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"storj.io/storj/internal/testcontext"
```

Comment:
```suggestion

	"storj.io/storj/internal/testcontext"
```

Fix linting error for import style